plugin.cfg

```diff
@@ -39,6 +39,7 @@ any:any
 chaos:chaos
 loadbalance:loadbalance
 cache:cache
+mdns:github.com/openshift-metal3/coredns-mdns
```

Comment:
Thought this would be an entire plugin :) We don't merge external plugins like this. It might be worth while to file an PR on coredns/coredns.io to list this there as an external plugin (I could do that as well - if you like)